opensrp-chw-referral/build.gradle

```diff
@@ -53,21 +53,82 @@ android {
     testOptions {
         unitTests {
             includeAndroidResources = true
+            returnDefaultValues = true
         }
     }
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
+    dataBinding {
+        enabled = true
+    }
+
 }
 
 dependencies {
+    implementation('org.smartregister:opensrp-client-core:1.7.32-SNAPSHOT@aar') {
+        transitive = true
+        exclude group: 'com.github.bmelnychuk', module: 'atv'
+        exclude group: 'com.google.guava', module: 'guava'
+        exclude group: 'com.android.support', module: 'appcompat-v7'
+        exclude group: 'net.zetetic', module: 'android-database-sqlcipher'
+        exclude group: 'org.apache.httpcomponents', module: 'httpmime'
+    }
+
+    implementation('org.smartregister:opensrp-client-native-form:1.6.1-RTL-SNAPSHOT@aar') {
+        transitive = true
+        exclude group: 'com.android.support', module: 'recyclerview-v7'
+        exclude group: 'com.android.support', module: 'appcompat-v7'
+        exclude group: 'com.android.support', module: 'cardview-v7'
+    }
+
+    implementation('org.smartregister:opensrp-client-configurable-views:1.1.2-SNAPSHOT@aar') {
+        transitive = true
+        exclude group: 'org.smartregister', module: 'opensrp-client-core'
+        exclude group: 'org.smartregister', module: 'opensrp-client-native-form'
+        exclude group: 'com.android.support', module: 'appcompat-v7'
+    }
+
+    api "org.smartregister:android-p2p-sync:0.3.2-SNAPSHOT"
+
```

Comment:
@cozej4 Are these libraries being used?

Reply:
@ellykits Yes these libraries are used in various areas within the code. I extend some classes that require these modules

Reply:
Alright.